vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/SockJSSocket.java

```diff
@@ -157,6 +158,11 @@ default void close(int statusCode, String reason) {
    */
   String uri();
 
+  /**
+   *  @return the Vert.x-Web RoutingContext corresponding to this socket
+   */
+  RoutingContext webRoutingContext();
```

Comment:
can you just name it to `routingContext` ?

Reply:
done